pkg/minikube/cruntime/cruntime.go

```diff
@@ -25,6 +25,11 @@ import (
 	"k8s.io/minikube/pkg/minikube/out"
 )
 
+const (
+	// OpenFilesMax is the maximum number of open files allowed by container runtimes. Arbitrary, but commonly used.
+	OpenFilesMax = 1048576
```

Comment:
This is only used in a test, so it probably doesn't belong here.

Reply:
Removed.